utils/kubernetes/kubeutil.py

```diff
@@ -388,7 +388,7 @@ def get_deployment_for_replicaset(self, rs_name):
         if end > 0 and rs_name[end + 1:].isdigit():
             # k8s before 1.8
             return rs_name[0:end]
-        if end > 0 and len(rs_name[end + 1:]) == 10:
+        if end > 0 and len(rs_name[end + 1:]) > 2:
```

Comment:
If there's a dash in the replica set and at least 2 runes after it we'll assume it's a deployment. That's not great, but I don't see any other short-term fix either :/
Maybe we should document that (as a comment)? I don't expect this to be an issue, but we need to be able to point users at some doc if it is.

Reply:
At least three indeed, but that's a risk. I added a comment in the test, but I'll copy it in kubeutil too

Reply:
BTW for now if there's a dash and two numbers the old matching pattern will trigger too